app/src/app/pages/plando-page/plando-spirits-and-chapters/plando-spirits-and-chapters.component.ts

```diff
@@ -26,10 +29,28 @@ export class PlandoSpiritsAndChaptersComponent {
     { id: 'LavaPiranha', displayName: 'Lava Piranha' },
     { id: 'HuffNPuff', displayName: 'Huff N Puff' },
     { id: 'CrystalKing', displayName: 'Crystal King' }];
+  public spiritsSubscription: Subscription;
   public requiredSpiritsArray = Array<string>();
-
   public toDisplayString = pascalToVerboseString;
 
+  public ngOnInit(): void {
+    this.spiritsSubscription = this.plandoFormGroup.get('required_spirits').valueChanges.subscribe(val => {
+      if (!val) {
+        this.requiredSpiritsArray = [];
+      } else {
+        this.requiredSpiritsArray = val;
+      }
+    })
+  }
+
+  public ngAfterContentInit(): void {
+    this.requiredSpiritsArray = this.plandoFormGroup.get('required_spirits').value;
```

Comment:
This seems to break when there is no saved preset. I can't test without commenting it or the html crashes, seems like it loads null or undefined into it

Reply:
Whoops! Fixed.